auth/actiongroups.go

```diff
@@ -0,0 +1,121 @@
+package auth
+
+type (
+	ActionGroup struct {
+		Actions []string
+	}
+)
+
+var (
+	// APIs matche globalAPIResponses in https://github.com/temporalio/temporal/blob/main/common/rpc/interceptor/redirection.go
+	workflowServiceGlobalAPIs = []string{
```

Comment:
Proxy also needs to allow UpdateNamespace for setting active/passive clusters, and UpdateNamespace is part of workflow service.

Which methods are _not_ allowed here? Seems like we have most of them.

Reply:
Set of workflowservice APIs are not in the global (or forward) APIs
< DeprecateNamespace
< DescribeNamespace
< GetClusterInfo
< GetSearchAttributes
< GetSystemInfo
< ListNamespaces
< RegisterNamespace
< UpdateNamespace